test/NuGet.Core.FuncTests/Dotnet.Integration.Test/MsbuildIntegrationTestFixture.cs

```diff
@@ -32,6 +37,27 @@ public MsbuildIntegrationTestFixture()
 
             var sdkPaths = Directory.GetDirectories(Path.Combine(_cliDirectory, "sdk"));
 
+            //Temporary patching process for System.Security.Cryptography.Pkcs.dll and deps.json files
+            //Will be removed when shipping
+            var patchDir = sdkPaths.Where(path => path.Split(Path.DirectorySeparatorChar).Last().StartsWith("5")).First();
```

Comment:
Yuck.  Why `"5"`?

How about:
```C#
const string net5MajorVersion = "5";

...StartsWith(net5MajorVersion)…
```

Also replace `var` with the explicit type, since the explicit type is not immediately obvious.